package.json

```diff
@@ -30,8 +30,8 @@
     "build:es:watch": "NODE_ENV=development lerna run --no-private --parallel build:es:watch"
   },
   "workspaces": [
+    "application-templates/*",
```

Comment:
Quick q: why not `packages/application-templates` instead of opening a whole new workspace?

Reply:
I find it weird to mix it with "normal" packages

Reply:
I find it weird to make a whole new workspace just for one package. That makes me feel that this is actually something which does not really belong into the repo after all :).

Reply:
Why not? What is the problem of having different workspaces? It's just a way or organising things. The template is not really a library package but an app. We do the same in our apps repo.

The big advantage in having the template in the monorepo is that dep versions are automatically updated as we make changes. If we move it to a separate repo we need to manage those deps manually.